src/devices/Mcp23xxx/Mcp23S17SpiAdapter.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
file name should be Mcp23s17.Mcp23S17SpiAdapter.cs

Reply:
I renamed it.